tests/irepela/homework_4/test_list_methods.py

```diff
@@ -0,0 +1,28 @@
+import pytest
+
+some_str = "fabcdefdgab"
+some_list = [1, 2, 3, 4, 5, 6, 1, 3, 2, 5, 4]
+some_tuple = (1, 2, 3, 4, 5, 6, 1, 3, 2, 5, 4)
+
+
+@pytest.mark.parametrize("a, b, expected", [
+    ("a", some_str, 2),
+    (5, some_list, 2),
+    (6, some_tuple, 1),
+])
+def test_list_count(a, b, expected):
+    assert b.count(a) == expected
+
+
+@pytest.mark.parametrize("a, b, expected", [
+    ("c", some_str, 3),
+    (6, some_list, 5),
+    (6, some_tuple, 5),
+])
+def test_list_index(a, b, expected):
+    assert b.index(a) == expected
+
+
+def test_list_extend():
+    some_list.extend([7, 8, 9])

```

Comment:
Here you break isolation between tests, because you modify global object, DONT DO THAT
since now your tests depend on run order